doc/extension_store/README.md

```diff
@@ -78,3 +83,11 @@ The Extension management component on the cluster must store the following infor
     * Easy solution: user has to completely delete the VS and create it from scratch
 * How to handle rollbacks in case installation fails?
 * Revert to previous version if an upgrade fails?
+* How to detect if an adapter is still used when trying to delete it?
+    * Maybe configure a script that checks if it is still in use.
+* Idea: Don't do updates of existing extensions, just install the new version and keep the old one.
+    * Upgrading an existing VS would require deleting and re-installing?
```

Comment:
So, this would require the user to reenter the credentials, except we can reuse the connection.

Reply:
Added a note about this.